libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -376,17 +379,25 @@ data FeatureLegalHold
   | FeatureLegalHoldDisabledByDefault
   deriving (Eq, Ord, Show, Enum, Bounded, Generic)
 
+data FeatureCustomSearchVisibility
+  = FeatureCustomSearchVisibilityEnabledByDefault
+  | FeatureCustomSearchVisibilityDisabledByDefault
+  deriving (Eq, Ord, Show, Enum, Bounded, Generic)
```

Comment:
Looks like that commit went away before I got a chance to look at it. I think we shouldn't keep doing bad things while waiting for us to move to servant. This kind of stuff doesn't have to have its own issue, if it does, it will probably never get prioritized. Instead we can just incrementally refactor it out of our code.
I am not sure how servant will solve this, but I am also not very hopeful that we'll get time to move to servant so soon.

> Can we open an issue on that and let it slip through for now?

If this is time sensitive, I guess it is not too much debt to be taking on.

Reply:
https://github.com/zinfra/backend-issues/issues/1407

I think this is worth its own issue just for the discussion.

> If this is time sensitive, I guess it is not too much debt to be taking on.

I would appreciate that.  :)

But if you make it go away here I won't fight you.